docs/user_guide/command_line_interface.rst

```diff
@@ -513,10 +514,11 @@ The following command shows how to use ``marius_postprocess`` for retrieving tra
 
 ::
 
-    marius_postprocess ./data/ ./dataset_directory --output_directory output_dir -f CSV
+    marius_postprocess ./data/marius/ ./dataset_directory --output_directory output_dir -f CSV
 
-In this case, ``./data/`` is the directory created by ``marius_train`` containing all the
-trained embeddings. ``./dataset_directory`` is the directory created by ``marius_preprocess``
+In this case, ``./data/marius/`` is the directory created by ``marius_train`` containing all the
+trained embeddings (the default ``general.experiment_name`` is set to ``marius``). 
```

Comment:
Remove the part in the parenthesis. 

"In this case, ``./data/marius/`` is the directory created by  ``marius_train`` which contains the trained model."